compiler/IREmitter/Payload/patches/vm_insnhelper.c

```diff
@@ -977,3 +977,16 @@ VALUE sorbet_vm_instance_variable_get(struct FunctionInlineCache *getCache, IVC
     reg_cfp->sp += 2;
     return sorbet_callFuncWithCache(getCache, VM_BLOCK_HANDLER_NONE);
 }
+
+VALUE sorbet_vm_class(struct FunctionInlineCache *classCache, rb_control_frame_t *reg_cfp, VALUE recv) {
+    sorbet_vmMethodSearch(classCache, recv);
```

Comment:
I think since this is the only intrinsic for `class`, we don't have to worry about this -- but this is a good thing to keep in mind for future intrinsics!